FlightReservation/components/termsComponents.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  TextInput,
+  Pressable,
+  Image,
+  TouchableOpacity,
+} from 'react-native';
+import CheckBox from '@react-native-community/checkbox';
+
+
+function Terms(props) {
+    const { changeForm, objValues } = props;
+    return (
+      <View style={[styles.containerForm, { marginTop: 30 }]}>
```

Comment:
indentation here has 4 spaces, but inside the JSX it has 2 spaces. Make a consistent spacing among the project is importante